.github/labeler.yml

```diff
@@ -0,0 +1,13 @@
+# changes to documentation
+"area/documentation": doc/**/*
+
+# changes to the core lib package
+"area/lib": ./*.go
+
+# changes to the zsh completion
+"area/*sh completion":
+  - ./zsh_*
+  - ./shell_*
+  - ./powershell_*
+  - ./bash_*
```

Comment:
This should also match `fish_completions*` and `
custom_completions*`. Maybe it is easier to just match on the `completions` substring.

Reply:
True - ended up just globing for "completions". Thanks for this!!